libminifi/src/Processor.cpp

```diff
@@ -226,6 +250,39 @@ bool Processor::setProperty(std::string name, std::string value)
 	}
 }
 
+bool Processor::setDynamicProperty(std::string name, std::string value)
+{
+	std::lock_guard<std::mutex> lock(_mtx);
+	std::map<std::string, Property>::iterator it = _dynamicProperties.find(name);
```

Comment:
Just a general comment about iterators. We could make this simpler with C++11 type deduction:
auto it = _dynamicProperties.find(name).